core/Cargo.toml

```diff
@@ -108,6 +108,9 @@ anyhow = "1"
 assert_matches = "1.5"
 async-trait = "0.1"
 async-recursion = "1"
+aws-config = { version = "1.1.7", features = ["behavior-version-latest"] }
+aws-runtime = "1.5.5"
+aws-sdk-s3 = "1.76.0"
```

Comment:
Could you please check whether all default features are required for these crates to work? E.g., `aws-config` has the `sso` feature enabled by default bringing additional deps, which looks unused; at least the code compiles with

```toml
aws-config = { version = "1.1.7", default-features = false, features = ["behavior-version-latest"] }
```

Reply:
Perfect idea, removed 2 dependencies.